pubspec.yaml

```diff
@@ -25,6 +25,8 @@ dependencies:
   provider: ^3.1.0+1
   flutter_slidable: ^0.5.4
   fluttertoast: ^3.1.3
+  location: ^2.3.5
+  flutter_compass: ^0.3.4
```

Comment:
Is this stand alone or does it drag in other dependancies?

Reply:
The plugins bring in whatever dependencies they depend on, I don't think its a problem because it includes whatever version of the dependencies they were published with. It is good to know though in case we need to update packages.

Reply:
I am more concerned with permissions/ data collections/ etc. Are these plugins open source? I know that we are trying to keep the communications/ locations of our users private so we should ensure our dependencies uphold that effort.

Reply:
For reference the dependencies can be seen on the sidebar of the plugins and they can be found here: https://pub.dev/packages

Reply:
They're all open source, I haven't looked into the data collection side of them.